keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -369,6 +371,43 @@ of the following scenarios:
 2. A new Node results in a Service that is able to achieve an endpoint
    distribution below 20% for the first time.
 
+### Manual enforcement of Topology Aware Routing by zone
+
+The algorithm proposed for mitigating the risk of overload works best with large
+numbers of endpoints and nodes, such that the percentage thresholds are unlikely
+to be broached with service restarts and node additions/removals within zones.
+There are a number of use cases however where the proposed algorithm for automatic
+assignments of zone hints is not suitable:
+
+1. Small multi-zone clusters with a low number of zones and endpoints. For 
+example a rolling restart with a service in a 3 zone cluster with 2-2-2 
+endpoint distribution will cause the proposed algorithm to remove hints for the 
+service during the restart, this could have severe impact for high latency 
+environments.
+2. Multi-zone clusters that deploy applications in a subset of zones. Unless 
+all zones in a cluster contain an endpoint then Topology Hints will not be 
+applied.
```

Comment:
is this not a deployment problem? 

if you create a multizone deployment I expect the user wants to provide high availability, leaving one zone without endpoints sounds like a flaw in the design, or am I missing something?

I mean, the network should be used to improve the solution, not to workaround it :)